build.gradle.kts

```diff
@@ -59,10 +59,14 @@ tasks {
     }
 
     shadowJar {
+        exclude("/prebuilt/linux/aapt")
+        exclude("/prebuilt/linux/aapt_64")
+        exclude("/prebuilt/macosx/aapt_64")
+        exclude("/prebuilt/windows/aapt.exe")
+        exclude("/prebuilt/windows/aapt_64.exe")
         minimize {
-            exclude(dependency("org.jetbrains.kotlin:.*"))
             exclude(dependency("org.bouncycastle:.*"))
-            exclude(dependency("app.revanced:.*"))
```

Comment:
ReVanced Library is used, why is only patcher excluded?

Reply:
revanced-library does not need to be excluded from minimization.

Patcher classes may be referenced by the patches jar, so all classes must be kept.
On the other hand, ReVanced Library is only used by CLI code and is referenced statically, so there is no need to keep it.
This is the same reason why you don't need to exclude pico-cli dependency.

I tested patching, list patches, list versions, mount/unmount, and install/uninstall. It worked as expected.